ddtrace/tracer/option.go

```diff
@@ -161,3 +161,10 @@ func WithError(err error) FinishOption {
 		cfg.Error = err
 	}
 }
+
+// Disable backtrace generation in case of error.
```

Comment:
You'll have to phrase this starting the name of the identifier, otherwise it's not standard and the linter will warn you:
```go
// NoDebugStack prevents any error presented using the WithError finishing option
// from generating a stack trace. This is useful in situations where errors are frequent
// and performance is critical.
```